modules/ting_das/ting_das.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * @file
+ * Handle the integration of digital article service (DAS) into the CMS.
+ */
+
+/**
+ * Implements hook_das_menu().
+ */
+function ting_das_menu() {
+  $items = array();
+
+  $items['admin/config/ting/das'] = array(
+    'title' => 'Digital article service',
+    'description' => 'Configure access to DAS.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ting_das_admin_settings_form'),
+    'access arguments' => array('administer site configuration'),
+    'file' => 'includes/ting_das.admin.inc',
+  );
+
+  $items['das/%ting_object/order'] = array(
+    'page callback' => 'ting_das_ajax',
+    'page arguments' => array(1),
+    'delivery callback' => 'ajax_deliver',
+    'access arguments' => array('access content'),
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_libraries_info().
+ *
+ * For defining external libraries.
+ */
+function ting_das_libraries_info() {
+  return array(
+    'guzzle' => array(
```

Comment:
On its own this is not sufficient to use Guzzle. As I see it the preferred approach going forward is to depend on `ding_libs` (see c3e8bd0f46c0167fcc94ef5dd5ce260b36052497).

Reply:
yes, I known... ding_libs did not exits when I made this module 6-7 months ago. Will change it to use ding_libs.